sphinxcontrib/adadomain.py

```diff
@@ -568,9 +568,14 @@ def _find_obj(self, env, modname, name, objtype, searchorder=0):
     def resolve_xref(
         self, env, fromdocname, builder, typ, target, node, contnode
     ):
+        # Resolve classwide type references to their base type
+        real_target = target
+        if target.endswith("'Class"):
```

Comment:
Just in case: it is possible for `target` to be lowercase? (`'class`)

Reply:
No it's not possible. I'll add a comment